frontend/packages/knative-plugin/src/components/overview/FilterTable.tsx

```diff
@@ -39,21 +36,44 @@ const FilterTable: React.FC<FilterTableProps> = ({
       ],
     };
   };
+
+  const { t } = useTranslation('knative-plugin');
+
   const data = {
-    columns: ['Key', 'Value'],
+    columns: [t('Key'), t('Value')],
```

Comment:
Change `Key` to `Attribute` to make it consistent with Trigger form label for filter

```suggestion
    columns: [t('Attribute'), t('Value')],
```